.github/workflows/php-tests-wp-4-7-php-5-6.yml

```diff
@@ -0,0 +1,69 @@
+name: PHP Tests
+
+on:
+  push:
+    branches:
+      - develop
+      - master
+  pull_request:
+    types:
+      - opened
+      - reopened
+      - synchronize
+
+jobs:
+  php-tests-wp-4-7-php-5-6:
+    name: WP 4.7, PHP 5.6
+    runs-on: ubuntu-latest
+    services:
+      mysql:
+        image: mysql:5.7
+        env:
+          MYSQL_ALLOW_EMPTY_PASSWORD: yes
+        ports:
+          - 3306:3306
+        options: --health-cmd="mysqladmin ping" --health-interval=5s --health-timeout=2s --health-retries=3
+    env:
+      DB_HOST: 127.0.0.1
+      DB_PORT: 3306
+      MYSQL_USER: user
+      MYSQL_PASSWORD: password
+      MYSQL_DATABASE: wordpress_test
+      MYSQL_ROOT_PASSWORD: password
+      WP_VERSION: '4.7'
+    steps:
+      - uses: actions/checkout@v2
+      - name: Verify MySQL connection
+        run: |
+          while ! mysqladmin ping -h"$DB_HOST" -P"$DB_PORT" --silent; do
+            sleep 1
+          done
+      - name: Grant database access
+        env:
+          MYSQL_TCP_PORT: ${{ env.DB_PORT }}
+        run: |
+          mysql -u root -h ${DB_HOST} -e "CREATE USER ${MYSQL_USER} IDENTIFIED BY '${MYSQL_PASSWORD}';"
+          mysql -u root -h ${DB_HOST} -e "GRANT ALL PRIVILEGES ON *.* TO ${MYSQL_USER};"
+          mysql -u root -h ${DB_HOST} -e "FLUSH PRIVILEGES;"
```

Comment:
This is not needed.

Reply:
Trying to use the root user always results in an error for me, so I think we need this:

<img width="901" alt="Screenshot 2020-12-21 at 19 20 36" src="https://user-images.githubusercontent.com/90871/102814274-36f9f780-43c2-11eb-8e6a-ca798d59448d.png">